### ISSUE TITLE
not compatible with jQuery 3.x

### ISSUE THREAD

**Issue by SoulOfNoob** [jaydenseric/Purty-Picker#4]
"$spectrum.height()" in line 128 in color-picker.js has to be replaced with: "$spectrum.outerHeight()" in order to work with jQuery 3.x because it doesn't count padding to the height.

**Comment by jaydenseric** [comment]
Thanks for the report 🙂 

I likely won't get around to updating this project or testing any fixes for a long time; I had a new version in the works but have since moved on to React components and do things very differently these days. Also I don't have need of a color picker in any of my current projects.

Feel free to PR, but it won't get merged for a while.

**Comment by SoulOfNoob** [comment]
Yeah i already thought it would not be updated 😅 but i wanted to leave the solution somewhere just in case someone needs it :)

**Comment by drakanor** [comment]
Thanks for pointing that out. It happens in jQuery 2 (latest) as well.

Would be very cool to see this color picker getting an update or even an AngularJS implementation. I was searching for a color picker suited for mobile websites and have not found a better one yet in terms of display and usability.

**Comment by jaydenseric** [comment]
@drakanor thanks for the kind words :) It would be fun to write a new one with the same design in React; it would be really technically elegant. I'll get to it whenever I next have a project in need of color inputs; feel free to rip-off the design if you would like to take a crack at it in the mean time.

For other purposes I have come across a really amazing [`shadeBlendConvert` color manipulation function](https://stackoverflow.com/a/13542669/1596978) I would love to help the author publish to npm. It might be handy for a new Purty Picker implementation.
